src/e2e/java/teammates/e2e/util/BackDoor.java

```diff
@@ -333,9 +333,10 @@ public static CourseAttributes getArchivedCourse(String instructorId, String cou
     /**
      * Gets instructor data from the datastore.
      */
-    public static InstructorData getInstructorData(String courseId, String email) {
+    public static InstructorData getInstructorData(String courseId, String instructorId, String email) {
         Map<String, String[]> params = new HashMap<>();
         params.put(Const.ParamsNames.COURSE_ID, new String[] { courseId });
+        params.put(Const.ParamsNames.USER_ID, new String[] { instructorId });
```

Comment:
Do we still need this `USER_ID` parameter, now that you are guaranteed to be admin if using the backdoor method?